### PR TITLE
Allows us to run integration package unit tests by adding a PR label

### PR DESCRIPTION
**Pull request by chrisguidry**
There are times when we make changes to `prefect` that we suspect may affect
the integrations packages, but we don't want to run the full integrations test
suites on _every_ `prefect` change.  This lets us add a `test-all-integrations` label to
a PR to opt in to running those tests.

### CODE REVIEW

File: src/prefect/__init__.py [hunk 1]
```diff
@@ -104,3 +104,6 @@
     "resume_flow_run",
     "suspend_flow_run",
 ]
+
+
+# NO-OP CHANGE
```
Comment (zzstoatzz) [inline src/prefect/__init__.py:109]:
```suggestion
```

**Review by zzstoatzz — APPROVED** [review]
lgtm!

File: src/prefect/__init__.py (hunk 1 above)
Comment (chrisguidry) [reply src/prefect/__init__.py:109]: woops ty

File: .github/workflows/integration-package-tests.yaml [hunk 2]
```diff
@@ -3,7 +3,9 @@ name: Integrations Packages Tests
 on:
   pull_request:
     paths:
-      - "src/integrations/*/**"
+      - .github/workflows/integration-package-tests.yaml
+      - "src/**/*"
```
Comment (serinamarie) [inline .github/workflows/integration-package-tests.yaml:7]: was this meant to be only on `.py` files? I'm surprised that my changes to the `prefect-ray` readme are running this workflow
https://github.com/PrefectHQ/prefect/pull/13630/files

Comment (serinamarie) [reply .github/workflows/integration-package-tests.yaml:7]: for sure!

**Pull request merged**